templates/base/kernel-ci-base.jinja2

```diff
@@ -37,7 +37,7 @@ device_type: {{ base_device_type }}
 
 {% if context %}
 context:
-{% for key, value in context.items() %}  {{ key }}: "{{ value }}"
+{% for key, value in context.items() %}  {{ key }}: {{ value }}
```

Comment:
I can't remember why exactly, but I'm sure these quotes around the {{ value }} were for some context that was being set needing to be forced into being a string.
Was there a reason to remove them?

Reply:
because some values need to be without quotes, let me refind which ones

Reply:
In fact it was in the commit description, you need to be able to pass numeric values, and adding quotes breaks the numeric status

Reply:
FYI this has been tested on staging for days and there doesn't seem to be any issue without the quotes.